bin/download-translations.js

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env node
+
+// This script downloads translation files from the Genesis GlotPress language
+// server at https://translate.studiopress.com/global/projects/. It was ported
+// from grunt-glotpress: https://github.com/markoheijnen/grunt-glotpress.
+//
+// Type the `npm run translations` command to run it.
+//
+// The script expects options to appear under a glotpress key in package.json:
+//
+// "glotpress": {
+// 	"url": "https://translate.studiopress.com/global/",
+// 	"slug": "genesis/genesis",
+// 	"textdomain": "genesis",
+// 	"fileFormat": "lib/languages/%wpLocale%.%format%",
+// 	"formats": [
+// 		"po",
+// 		"mo"
+// 	],
+// 	"filters": {
+// 		"minimumPercentage": 30
+// 	}
+// },
+
+const axios = require( 'axios' );
+const fs = require( 'fs' );
+
+const package = require( process.cwd() + '/package.json' );
+const options = package.glotpress;
+
+axios.defaults.headers.common['User-Agent'] = `${package.glotpress.textdomain}`;
```

Comment:
A UA string that indicated it was from `Genesis Translations` plugin would be more indicative that just `genesis`, right?

Reply:
Good call — I changed it to `package.name`, which gives `genesis-translations`.